dashboards/kibana/kibana-prometheus.json

```diff
@@ -75,25 +107,31 @@
             },
             "dataSets": [
               {
+                "breakdowns": [],
+                "dimensions": [],
+                "measures": [],
                 "plotType": "LINE",
                 "targetAxis": "Y1",
                 "timeSeriesQuery": {
-                  "prometheusQuery": "rate(kibana_millis_uptime[5m])"
+                  "prometheusQuery": "kibana_millis_uptime{${Cluster},${Location},${Namespace}}",
+                  "unitOverride": ""
```

Comment:
Can we set unitOverride to "ms" and remove the unit from the title? That way if it's a number like 2000, the chart will automatically relabel the y axis to say "2s"